### PR TITLE
#1104first commit

### PR DESCRIPTION
**Pull request by jhansi1104**
## Project

dashboard builder

### Changes

weekly update-for week 1 & week 2

### CODE REVIEW

File: docs/2023/Farmstack/Dashboard Builder/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/Yaus/Dashboard Builder
+---
+
+
+# Author
+Rishabh Jain
```
Comment (Apoorvgarg-creator) [inline docs/2023/Farmstack/Dashboard Builder/index.md:9]: Change the name to Jhansi

**Review by Apoorvgarg-creator — CHANGES_REQUESTED** [review]

File: docs/2023/Farmstack/Dashboard Builder/index.md (hunk 1 above)
Comment (jhansi1104) [reply docs/2023/Farmstack/Dashboard Builder/index.md]: Updated!

File: docs/2023/Farmstack/Dashboard Builder/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/Yaus/Dashboard Builder
```
Comment (shubhigupta991) [inline docs/2023/Farmstack/Dashboard Builder/index.md:4]: change it to 
slug: /2023/Farmstack/Dashboard Builder

File: docs/2023/Farmstack/Dashboard Builder/updates/2023-07-21.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: Week 3
+author: Rishabh Jain
```
Comment (shubhigupta991) [inline docs/2023/Farmstack/Dashboard Builder/updates/2023-07-21.md:3]: In all the file change name to Jhansi

**Review by shubhigupta991 — CHANGES_REQUESTED** [review]

File: docs/2023/Farmstack/Dashboard Builder/index.md (hunk 2 above)
Comment (jhansi1104) [reply docs/2023/Farmstack/Dashboard Builder/index.md]: Done

File: docs/2023/Farmstack/Dashboard Builder/updates/2023-07-21.md (hunk 3 above)
Comment (jhansi1104) [reply docs/2023/Farmstack/Dashboard Builder/updates/2023-07-21.md]: Done

**Pull request merged**